packages/seal-redisearch-adapter/src/RediSearchSearcher.php

```diff
@@ -96,6 +97,23 @@ public function search(Search $search): Result
                     $filter->latitude,
                     ($filter->distance / 1000) . ' km',
                 ),
+                $filter instanceof Condition\GeoBoundingBoxCondition => ($filters[] = \sprintf(
+                    '@%s:[WITHIN $filter_%s]',
+                    $this->getFilterField($search->indexes, $filter->field),
+                    $key,
+                )) && ($parameters['filter_' . $key] = \sprintf(
+                    'POLYGON((%s %s, %s %s, %s %s, %s %s, %s %s))',
+                    $filter->westLongitude,
+                    $filter->northLatitude,
+                    $filter->westLongitude,
+                    $filter->southLatitude,
+                    $filter->eastLongitude,
+                    $filter->southLatitude,
+                    $filter->eastLongitude,
+                    $filter->northLatitude,
+                    $filter->westLongitude,
+                    $filter->northLatitude,
+                )),
```

Comment:
GEO type does not support yet filtering by Polygon. Open issue is here: https://github.com/RediSearch/RediSearch/issues/680. Maybe it for them easier to support the bounding box. Lets wait for feedback from them: https://github.com/RediSearch/RediSearch/issues/5032